Library/Homebrew/cask/lib/hbc/dsl/depends_on.rb

```diff
@@ -84,23 +84,23 @@ def cask=(*arg)
         @cask.concat(Array(*arg))
       end
 
-      def macos=(*arg)
+      def macos=(*args)
         @macos ||= []
-        macos = if arg.count == 1 && arg.first =~ /^\s*(<|>|[=<>]=)\s*(\S+)\s*$/
+        macos = if args.count == 1 && args.first =~ /^\s*(<|>|[=<>]=)\s*(\S+)\s*$/
           raise "'depends_on macos' comparison expressions cannot be combined" unless @macos.empty?
           operator = Regexp.last_match[1].to_sym
           release = self.class.coerce_os_release(Regexp.last_match[2])
           [[operator, release]]
         else
           raise "'depends_on macos' comparison expressions cannot be combined" if @macos.first.is_a?(Symbol)
-          [*arg].map(&self.class.method(:coerce_os_release)).sort
+          Array(*args).map(&self.class.method(:coerce_os_release)).sort
```

Comment:
I changed this to `[*arr]` today in the RuboCop PR, and after that I got errors locally.

I think this is due to the following scenario:

```ruby
# in cask

depends_on macos: ["10.9", "10.10"]

# inside `macos=`

args = [["10.9", "10.10"]]

Array(*args) == ["10.9", "10.10"]
[*args] == [["10.9", "10.10"]]

```